.github/ISSUE_TEMPLATE/issue-template-.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 'Issue template '
+about: Template for creating issues
+title: User story or short precise description
+labels: ''
+assignees: ''
+
+---
+
+##User story
+Write it from the user perspective (end-user, admin) if it's a feature.
+Write if from the developers (or equal) perspective, if it's a change to the code architechture or 
```

Comment:
Typo it should be "it" not "if" i think.

Reply:
Yes. Nice spotted.